sdk/textanalytics/Azure.AI.TextAnalytics.Legacy.Shared/src/autorest.md

```diff
@@ -0,0 +1,244 @@
+# Azure.AI.TextAnalytics.Legacy.Shared
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+### Old Swagger
+
+``` yaml
+input-file:
+    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/11cb5c629a836dc99454d85c233405f952b555d8/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.2-preview.2/TextAnalytics.json
+generation1-convenience-client: true
+namespace: Azure.AI.TextAnalytics.Legacy
+title: TextAnalyticsClient
+modelerfour:
+    seal-single-value-enum-by-default: true
+    lenient-model-deduplication: true
+```
+
+### Update definitions' namespace to Azure.AI.TextAnalytics.Legacy
+```yaml
+directive:
+  from: swagger-document
+  where: $.definitions.*
+  transform: >
+    $["x-namespace"] = "Azure.AI.TextAnalytics.Legacy"
+```
+
+### Update operations' namespace to Azure.AI.TextAnalytics.Legacy
+```yaml
+directive:
+  from: swagger-document
+  where: $.operations.*
+  transform: >
+    $["x-namespace"] = "Azure.AI.TextAnalytics.Legacy"
+
+```
+### Make types internal
+
+``` yaml
+directive:
```

Comment:
I went with the expanded form after discussing with Heath; functionally, its the same as having 3 separate sections.